spec/features/idv/hybrid_flow_spec.rb

```diff
@@ -11,14 +11,15 @@
       and_return(OpenStruct.new(region: 'us-west-2', account_id: '123456789'))
   end
 
-  it 'proofs and hands off to mobile', js: true do
-    user = nil
-    sms_link = nil
-
+  before do
```

Comment:
Seems odd to me to have an expectation in a before block.

Reply:
@solipet which do you think would be better?
1. switching to `allow` (the goal is to capture the SMS url, not necessarily an expectation)
2. having it inside both specs in this file?

Reply:
switched to `allow` in 2f3003ec18d4db0eec8c8fcb3112e59731d27ee8

Reply:
looks good - thanks!